src/App.js

```diff
@@ -4,17 +4,18 @@ import AddTodoForm from './AddTodoForm';
 
 //main function
 function App() {
```

Comment:
Please make sure that your indent levels are consistent. This makes the code easier to read.